Extensions/postarchive.js

```diff
@@ -1130,7 +1156,7 @@ XKit.extensions.postarchive = {
 		var blog_url = m_post.owner;
 
 		if (!blog_url) {
-			blog_url = window.location.href.split('%2F')[2].split('.')[0];
+			blog_url = window.location.href.split('/')[2].split('.')[0];
```

Comment:
I replaced it with something that will work for custom domains but there's a bigger issue with custom domains: XKit doesn't run in the context of the top frame so the whole message passing thing doesn't work.